plugin/ochttp/trace.go

```diff
@@ -126,13 +126,8 @@ func (t *traceTransport) CancelRequest(req *http.Request) {
 	}
 }
 
-func spanNameFromURL(prefix string, u *url.URL) string {
-	host := u.Hostname()
-	port := ":" + u.Port()
-	if port == ":" || port == ":80" || port == ":443" {
-		port = ""
-	}
-	return prefix + "." + host + port + u.Path
+func spanNameFromURL(u *url.URL) string {
+	return u.Path
```

Comment:
Will the path be sufficient here?

Requests from "localhost:443/foo", "opencensus.io:8888/foo", "opencensus.io/foo", "ultima.te:9999/foo" will all have the same span name.

Reply:
Users should always breakdown by host. Unlike RPC, there is no concept of a service name in HTTP URLs. The hostname might differ even if the service is the same.